lib/css/components/inputs.less

```diff
@@ -462,21 +462,23 @@ fieldset {
 .s-input,
 .s-textarea,
 .s-select > select {
```

Comment:
> **Note**
> The `readonly` attribute does not apply to `select` elements. This style was pre-existing and won't hurt anything to include, but I was a little confused, so I'm making a note.

From [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/readonly):
> The attribute is not supported or relevant to [`<select>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/select) 

The [section detailing the differences between `readonly` and `disabled`](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/readonly#attribute_interactions) is worth a read too:
> The difference between [disabled](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/disabled) and readonly is that read-only controls can still function and are still focusable, whereas disabled controls can not receive focus and are not submitted with the form and generally do not function as controls until they are enabled.

Reply:
I can also drop a note in a comment in the file. I did test the readonly styles on selects (because why not), but they looked very "meh", so I dropped them from the `.is-readonly` rule below since they aren't technically supported anyways. Good callout